README.md

```diff
@@ -56,3 +56,7 @@ Do projektu backendowego został podpięty [springdoc](https://springdoc.org/#In
 - Spełnia kryteria akceptacyjne
 - Przeszło przez Code Review (poprawki zostały zaakceptowane)
 - Zostało dołączone do gałęzi develop (merge)
+
+## Wykres spalania:
+https://docs.google.com/spreadsheets/d/1tHKVENLM-GykhG75kQ3YZRpPJ57ifT13/edit?usp=sharing&ouid=105439589476939159664&rtpof=true&sd=true
```

Comment:
Wrzuć to jako:
```
Link do wykresu znajduje się [tutaj](https://docs.google.com/spreadsheets/d/1tHKVENLM-GykhG75kQ3YZRpPJ57ifT13/edit?usp=sharing&ouid=105439589476939159664&rtpof=true&sd=true)
```
Wtedy w redme będzie to wyglądało tak:
Link do wykresu znajduje się [tutaj](https://docs.google.com/spreadsheets/d/1tHKVENLM-GykhG75kQ3YZRpPJ57ifT13/edit?usp=sharing&ouid=105439589476939159664&rtpof=true&sd=true)
Link nie będzie bezpośrednio widoczny i będzie ładniej